### PR TITLE
hida tab in form if all params are expert and normal mode is active.

### LINKED ISSUE #1558: hide Section Tab(form) if all options are advanced
Now, it is impossible hide a section even if  expertLevel is set to LEVEL_ADVANCED.

### PR DESCRIPTION
**Pull request by pconesa**
Closes #1558

### CODE REVIEW

File: pyworkflow/gui/form.py [hunk 1]
```diff
@@ -1899,9 +1901,19 @@ def _checkChanges(self, paramName):
     def _checkAllChanges(self):
         for paramName in self.widgetDict:
             self._checkCondition(paramName)
-            
+
+    def _checkTabsVisibility(self):
+        for index, sectionW in enumerate(self._sections):
+            tabs = sectionW.master
+            if sectionW.section.isExpert():
```
Comment (delarosatrevin) [inline pyworkflow/gui/form.py]: @pconesa I don't want to add more work on you...but would be good to make this more consistent with the visibility of the parameters. What I mean, is that you should check not only for the Expert value, but also for the condition (same as for params) to decide when to show/hide a tab. 

Nice work anyway!

File: pyworkflow/gui/form.py [hunk 2]
```diff
@@ -1899,9 +1901,19 @@ def _checkChanges(self, paramName):
...
+                if self.protocol.expertLevel.get() == params.LEVEL_NORMAL:
```
Comment (delarosatrevin) [inline pyworkflow/gui/form.py]: .get() is not needed for comparision

**Comment by delarosatrevin** [comment]
Yes (I guess this is checked anyway, we should avoid double evaluation) and also if there is a condition for the Tab itself.

**Comment by pconesa** [comment]
I'm closing this until I've time to implement missing things.

**Pull request closed without merging**